libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/roadobject/incident/IncidentInfo.kt

```diff
@@ -38,6 +38,7 @@ import java.util.Date
  * @param multilingualAffectedRoadNames affected road names in different languages.
  *  May contain affected road names from [affectedRoadNames] property in case language is known for them.
  *  Key is a ISO 639-1, 2 letter language code, value - the list of affected road names.
+ *  @param length length of the incident.
```

Comment:
Do we have docs describing the units? Meters? Indices? Something else?

Reply:
It's in meters, I'll add to docs, thank you!